snapshot/src/main/java/io/zeebe/snapshots/impl/SnapshotChunkUtil.java

```diff
@@ -9,18 +9,34 @@
 
 import io.zeebe.snapshots.SnapshotChunk;
 import java.io.IOException;
+import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.zip.CRC32C;
+import java.util.zip.Checksum;
 
 final class SnapshotChunkUtil {
 
   private SnapshotChunkUtil() {}
 
-  static long createChecksum(final byte[] content) {
-    final CRC32C crc32 = new CRC32C();
-    crc32.update(content);
-    return crc32.getValue();
+  static long createChecksum(final String chunkName, final byte[] content) {
```

Comment:
Does this mean the individual chunk checksum now also include the filename? I think it's only necessary when computing the folder checksum - if we just want the file checksum the filename isn't necessary to include, imo.

Reply:
I was not sure if we wanted it. I thought we discussed it to unify the checksum calculations. I will revert these.